RayLin.md

```diff
@@ -1,33 +1,27 @@
-# 07/25 ~ 07/29
+# 08/01 ~ 08/05
 
 - [Video Control]
+  - Bug 1291013 - Closed-caption button spacing is asymetric
+    - review cancelled, :jaws is overloaded
+    - ask again after flag removed
```

Comment:
got it. mozreview refused to push if the assigned reviewer blocks review flag. I'll see who can review it. Thanks.